gnome-image-installer/pages/install/gis-scribe.c

```diff
@@ -1147,9 +1149,18 @@ gis_scribe_tee_thread (GTask            *task,
           g_prefix_error (&error, "error writing image to self: ");
           break;
         }
+
+      bytes_teed += r;
```

Comment:
I realised that 2 out of the 3 error paths above are not reached in the test suite, so I'll take a look at fixing that…

Reply:
The read and write-to-GPG paths are now tested (and work).